README.md

```diff
@@ -19,11 +19,21 @@
 1. [Development](#development)
 
 ## Usage
+Complete the following step ONLY before starting the application for the first time:
+1. Open MySql from the command line.
+2. Run the following command:
 
-> Pleaes run the following "package.json" scripts to start the application;<br>
+```sh
+CREATE DATABASE goodreads;
```

Comment:
You may be able to cook this in in your next PR / commit - looks like sequelize has no way to create a DB, but you can run a raw query as part of your seed script: https://stackoverflow.com/a/46694727

Your syntax will be something like `CREATE DATABASE IF NOT EXISTS goodreads`

Reply:
Yes, I fully agree. Thanks! I did find some solutions but I wanted to make sure my service was ready ASAP for you and Jamal.